packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/documents_batch/data_triggers/triggers/withdrawals/v0/mod.rs

```diff
@@ -104,8 +104,8 @@ pub fn delete_withdrawal_data_trigger_v0(
         .get_integer("status")
         .map_err(ProtocolError::ValueError)?;
 
-    if status != withdrawals_contract::WithdrawalStatus::COMPLETE as u8
-        || status != withdrawals_contract::WithdrawalStatus::EXPIRED as u8
+    if status != withdrawals_contract::v0::WithdrawalStatus::COMPLETE as u8
+        || status != withdrawals_contract::v0::WithdrawalStatus::EXPIRED as u8
```

Comment:
Withdrawal status should not be in a versioned contract, as we are not going to have a new list per version.

Reply:
I was thinking to reexport if no changes. So we keep it unversioned.